pom.xml

```diff
@@ -7,8 +7,9 @@
     <groupId>org.sqlflow</groupId>
     <artifactId>jsqlflow</artifactId>
     <packaging>jar</packaging>
-    <version>1.0-SNAPSHOT</version>
+    <version>0.0.1</version>
```

Comment:
maybe the version start from `0.1.0`?

Reply:
Agree.
The `pb` is being iterated. Let's update the version to `0.1.0` when the protocol is stabled.